sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/GroupByKeyTest.java

```diff
@@ -186,6 +176,112 @@ public void testCombiningAccumulatingProcessingTime() throws Exception {
       p.run();
     }
 
+    /**
+     * Tests that data from a processing time trigger flows through subsequent GroupByKey
+     * transforms. To test this with TestStream, we check that it arrives in an early pane,
```

Comment:
I don't see where you check it is in the early pane in the test. If it's there maybe a comment would help.

Reply:
In fact I had pushed this with it commented out. The PAssert itself only inspects early panes now.